inference-engine/tests/functional/plugin/shared/include/single_layer_tests/concat_4D.hpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2019 Intel Corporation
```

Comment:
I don't understand, why you create a duplicate of `concat`? If you needed it, you could extend existed `concat` tests. Could you do it? And one more comment again. Please add me to the review, if you change ie_test_infra and shared tests.